### DISCUSSION TITLE
Backstage Plugin for Cluster API

### DISCUSSION THREAD

**Discussion by voor** [kubernetes-sigs/cluster-api#6998] [Ideas]
This is to discussion and see if anyone is interested in a Cluster API capability into [Backstage](https://backstage.io/), which is also CNCF incubating project.

**Comment by alexbarbato** [comment]
In theory the lift is not that high to allow registration of a management cluster (much like the recently added EKS processor which effectively registers an AWS account) - simply being able to view clusters under management could be pretty useful

**Reply by alexbarbato** [reply]
https://github.com/backstage/backstage/pull/11789

**Comment by vrabbi** [comment]
this would be a great plugin in my opinion and could be really helpful. We use backstage a lot and would love to see a CAPI plugin!
